dev/com.ibm.ws.microprofile.openapi/resources/com/ibm/ws/microprofile/openapi/impl/validation/resources/ValidationMessages.nlsprops

```diff
@@ -15,83 +15,83 @@
+callbackInvalidPathItem=The Callback object must contain a valid path item. The path item value associated with key \"{0}\" is not a valid value
 
-contactInvalidURL=The Contact object must contain a valid URL. The \"{0}\" value specified for the URL is not valid.
-contactInvalidEmail=The Contact object must contain a valid email address. The \"{0}\" value specified for the email address is not valid.
+contactInvalidURL=The Contact object must contain a valid URL. The \"{0}\" value specified for the URL is not valid
+contactInvalidEmail=The Contact object must contain a valid email address. The \"{0}\" value specified for the email address is not valid
 
-exampleOnlyValueOrExternalValue=The Example object \"{0}\" specifies both \"value\" and \"externalValue\" fields. Specify only one of the fields.
+exampleOnlyValueOrExternalValue=The Example object \"{0}\" specifies both \"value\" and \"externalValue\" fields. Specify only one of the fields
 
-externalDocumentationInvalidURL=The ExternalDocumentation object must contain a valid URL. The \"{0}\" value specified for the URL is not valid.
+externalDocumentationInvalidURL=The ExternalDocumentation object must contain a valid URL. The \"{0}\" value specified for the URL is not valid
 
-infoTermsOfServiceInvalidURL=The Info object must contain a valid URL. The \"{0}\" value specified for \"termsOfService\" is not a valid URL.
-invalidUrl=The \"{0}\" object must contain a valid URL. The \"{1}\" value specified for the URL is not valid.
-invalidUri=The \"{0}\" object must contain a valid URI. The \"{1}\" value specified for the URI is not valid.
+infoTermsOfServiceInvalidURL=The Info object must contain a valid URL. The \"{0}\" value specified for \"termsOfService\" is not a valid URL
+invalidUrl=The \"{0}\" object must contain a valid URL. The \"{1}\" value specified for the URL is not valid
+invalidUri=The \"{0}\" object must contain a valid URI. The \"{1}\" value specified for the URI is not valid
 
-headerExampleOrExamples=The Header object \"{0}\" specifies both an example object and an examples object. Only one of them should be specified.
-headerSchemaAndContent=The Header object \"{0}\" MUST contain either a schema property, or a content property, but not both.
-headerSchemaOrContent=The Header object \"{0}\" MUST contain either a schema property or a content property.
-headerContentMap=The \"content\" map within header object \"{0}\" must only contain one entry.
+headerExampleOrExamples=The Header object \"{0}\" specifies both an example object and an examples object. Only one of them should be specified
+headerSchemaAndContent=The Header object \"{0}\" MUST contain either a schema property, or a content property, but not both
+headerSchemaOrContent=The Header object \"{0}\" MUST contain either a schema property or a content property
+headerContentMap=The \"content\" map within header object \"{0}\" must only contain one entry
```

Comment:
Is there any reason why "header" is not capitalized here? I think we should be consistent throughout, either capitalizing all or them or none of them.
Also, \"{0}\" should come before "header object."
Also, "only" should come after "contain."
Suggested change: "within \"{0}\" header object must contain only one entry"